src/CommandingOfficers/Strong.java

```diff
@@ -117,45 +121,40 @@ public void modifyUnitAttackOnUnit(BattleParams params)
     private static final int STRONGARM_BUFF = 10;
     private static final int STRONGARM_FOOT_BUFF = 20; // On top of STONGARM_BUFF
 
-    CODamageModifier damageMod = null;
-    CODefenseModifier defenseMod = null;
-    CODamageModifier damageModTroop = null;
-
-    StrongArmAbility(Commander commander)
+    StrongArmAbility()
     {
-      super(commander, STRONGARM_NAME, STRONGARM_COST);
-
-      damageMod = new CODamageModifier(STRONGARM_BUFF);
-      defenseMod = new CODefenseModifier(STRONGARM_BUFF);
-      damageModTroop = new CODamageModifier(STRONGARM_FOOT_BUFF);
-      for( UnitModel model : commander.getAllModels(UnitModel.TROOP) )
-      {
-        damageModTroop.addApplicableUnitModel(model);
-      }
+      super(STRONGARM_NAME, STRONGARM_COST);
     }
 
     @Override
-    protected void perform(MapMaster gameMap)
+    protected void enqueueCOMods(Commander myCommander, MapMaster gameMap, ArrayList<COModifier> modList)
     {
-      // Grant the base firepower/defense bonus.
-      myCommander.addCOModifier(damageMod);
-      myCommander.addCOModifier(defenseMod);
-      myCommander.addCOModifier(damageModTroop);
+      GenericCOModifier damageMod = new CODamageModifier(STRONGARM_BUFF);
+      GenericCOModifier defenseMod = new CODefenseModifier(STRONGARM_BUFF);
+      GenericCOModifier damageModTroop = new CODamageModifier(STRONGARM_FOOT_BUFF);
```

Comment:
Because the filtered damage modifier originally stored a list of `UnitModel`s it wanted to modify, and I wanted that to be ephemeral (so that you could apply it to several COs, then remove it from all of them at once)

Reply:
Big future plans, eh? I'd wait until then to change how things work, just in case they need to work in a different way than you envision right now. At this point it's just unnecessary code churn.

Reply:
I did actually have it set up that way at one point in development.
That said, the new modifiers are even less stateful than the old ones, so I can definitely move 'em back.